packages/forms/src/UIForm/utils/validation.js

```diff
@@ -116,7 +116,7 @@ export function validateSimple(
 
 	results[key] = validateValue(mergedSchema, value, properties, customValidationFn);
 
-	if (deepValidation && items) {
+	if (deepValidation && items && value) {
```

Comment:
what if value is `false` ?

Reply:
We are talking about deep validation. You can't deep validate a boolean